python/setup.py

```diff
@@ -21,12 +21,13 @@ def find_protoc_exec():
     found_args, rest_args = parser.parse_known_args(sys.argv)
     sys.argv = rest_args
     result = vars(found_args).get(argument_prefix, None)
+    if os.path.exists(result):
+        return result
```

Comment:
@sashafrey One doesn't simply evaluate `os.path.exists(None)`.
Maybe you want to write `if result and os.path.exists(result):`?
Anyway, I don't understand the reason to refactor this part.